src/components/Images.vue

```diff
@@ -0,0 +1,283 @@
+<!--
+  - This is the Images table in the K8s page.
+  -->
+<template>
+  <div>
+    <div v-if="k8sIsRunning" ref="fullWindow">
+      <Checkbox
+        :disabled="showImageManagerOutput"
+        :value="showAll"
+        label="Show all images"
+        @input="handleShowAllCheckbox"
+      />
+      <SortableTable
+        :headers="headers"
+        :rows="rows"
+        key-field="key"
+        default-sort-by="imageName"
+        :table-actions="false"
+        :paging="true"
+      >
+        <template #row-actions="{row}">
+          <div v-if="hasDropdownActions(row)">
+            <ButtonDropdown
+              :disabled="showImageManagerOutput"
+              :dropdown-options="buttonOptions(row)"
+              button-label="..."
+              size="sm"
+              @click-action="(rowOption) => doClick(row, rowOption)"
+            />
+          </div>
+          <div v-else></div>
+        </template>
+      </SortableTable>
+
+      <hr>
```

Comment:
Design feedback:
- The label (`Name of image to {pull|build}`) probably should be an `<label>` element, such that clicking on it automatically focuses the correct text box.
- It might look better to put the buttons to the right of the text box (in one line).

Reply:
figured it out

Reply:
I'm thinking something like:
```html
<div class="actions">
  <label /> <input /> <button />
  <label /> <input /> <button />
</div>
```
```css
  .actions {
    display: grid;
    grid-template-columns: 1fr auto;
  }
  .actions label {
    grid-column-start: 1;
    grid-column-end: 3;
    margin-top: 0.75em;
  }
  .actions input {
    width: 100%;
  }
  .actions button {
    margin-left: 0.75em;
  }
```
So that it looks something like:
![image](https://user-images.githubusercontent.com/3977982/115465877-1a40c580-a1e4-11eb-8688-3bf58fcf1309.png)

(Such that the buttons are aligned)